### PR TITLE
Provide German translation for the tutorial. (#640)

### PR DESCRIPTION
**Pull request by intergalacticsuperhero**
This is my first pull request ever. Please tell me if everything is alright or let me know what to change.
Thanks for your work :-)

### CODE REVIEW

**Comment by Cyprex** [comment]
I can check these if you need a german player to look these over.

File: core/assets/bundles/bundle_de.properties [hunk 1]
```diff
@@ -866,26 +866,26 @@ unit.chaos-array.name = Chaos Array
...
-tutorial.blockinfo = Each block has different stats. Each drill can only mine certain ores.\nTo check a block's info and stats,[accent] tap the "?" button while selecting it in the build menu.[]\n\n[accent]Access the Mechanical Drill's stats now.[]
-tutorial.conveyor = [accent]Transportbänder[] werden dazu benutzt Materialien zum Kern zu transportieren.\n Erstelle eine Reihe von Transportbändern zum Kern.
-tutorial.conveyor.mobile = [accent]Conveyors[] are used to transport items to the core.\nMake a line of conveyors from the drill to the core.\n[accent] Place in a line by holding down your finger for a few seconds[] and dragging in a direction.\n\n[accent]{0}/{1} conveyors placed in line\n[accent]0/1 items delivered
+tutorial.next = [lightgray]<Tippen um fortzufahren>
+tutorial.intro = Du befindest dich im[scarlet] Mindustry-Tutorial.[]\nBeginne, indem du[accent] Kupfer abbaust[]. Tippe dazu auf ein Kupfervorkommen in der Nähe deiner Basis.\n\n[accent]{0}/{1} Kupfer
+tutorial.drill = Manuelles Abbauen von Ressourcen ist ineffizient.\n[accent]Bohrer []können automatisch abbauen.\nPlatziere einen Bohrer auf einem Kupfervorkommen.
```
Comment (Cyprex) [inline core/assets/bundles/bundle_de.properties]: Second Bohrer might need to have the [accent] as well, not sure how it's done in english

File: core/assets/bundles/bundle_de.properties [hunk 2]
```diff
@@ -866,26 +866,26 @@ unit.chaos-array.name = Chaos Array
...
+tutorial.drill.mobile = Manuelles Abbauen ist ineffizient.\n[accent]Bohrer []können automatsich abbauen.\nTippe auf den Bohrer Tab unten rechts.\nWähle den[accent] Mechanischen Bohrer[].\nPlatziere ihn durch Tippen auf ein Kupfervorkommen, dann klicke auf das[accent] Häkchen[] unten um deine Auswahl zu bestätigen.\nKlicke auf den [accent] X-Buttton[] um den Bau abzubrechen.
+tutorial.blockinfo = Jeder Block hat unterschiedliche Eigenschaften. Bohrer können immer nur eine Art von Material abbauen.\nFür Infos und Eigenschaften eines Blocks wähle einen Block im Baumenü aus und [accent] klicke auf den "?"-Button.[]\n\n[accent]Schau dir jetzt die Eigenschaften des Mechanischen Bohrers an.[]
```
Comment (Cyprex) [inline core/assets/bundles/bundle_de.properties]: Jeder Bohrer kann nur bestimmte Materialien abbauen.

Comment (Cyprex) [reply core/assets/bundles/bundle_de.properties]: Maybe Eigenschaften->Stats

File: core/assets/bundles/bundle_de.properties [hunk 3]
```diff
@@ -866,26 +866,26 @@ unit.chaos-array.name = Chaos Array
...
+tutorial.conveyor = [accent]Förderbänder[] werden dazu benutzt Materialien zum Kern zu transportieren.\nErstelle eine Reihe von Förderbändern zum Kern.
```
Comment (Cyprex) [inline core/assets/bundles/bundle_de.properties]: >Förderbänder/Transportbänder
Should be whatever "Titanium Conveyor" is translated to.

File: core/assets/bundles/bundle_de.properties [hunk 4]
```diff
@@ -866,26 +866,26 @@ unit.chaos-array.name = Chaos Array
...
+tutorial.conveyor.mobile = [accent]Förderbänder[] werden dazu benutzt Materialien zum Kern zu transportieren.\nErstelle eine Reihe von Förderbändern zum Kern.\n[accent] Zur Erstellung einer Reihe lasse deinen Finger für einige Sekunden gedrückt[] und ziehe ihn in eine beliebige Richtung.\n\n[accent]{0}/{1} Förderbänder in einer Reihe erstellt\n[accent]0/1 Ressourcen transportiert.
```
Comment (Cyprex) [inline core/assets/bundles/bundle_de.properties]: >Zur Erstellung einer Reihe lasse deinen Finger für einige Sekunden gedrückt[] und ziehe ihn in eine beliebige Richtung.

I find this sentence difficult to understand, might need rewording. Can't think of a better word than "Reihe" though.

File: core/assets/bundles/bundle_de.properties [hunk 5]
```diff
@@ -866,26 +866,26 @@ unit.chaos-array.name = Chaos Array
...
-tutorial.deposit = Deposit items into blocks by dragging from your ship to the destination block.\n\n[accent]Deposit your copper back into the core.[]
-tutorial.waves = Der [LIGHT_GRAY] Gegner[] greift an.\n\nVerteidige deinen Kern 2 Wellen lang. Bau mehr Türme.
-tutorial.waves.mobile = The[lightgray] enemy[] approaches.\n\nDefend the core for 2 waves. Your ship will automatically fire at enemies.\nBuild more turrets and drills. Mine more copper.
-tutorial.launch = Once you reach a specific wave, you are able to[accent] launch the core[], leaving your defenses behind and[accent] obtaining all the resources in your core.[]\nThese resources can then be used to research new technology.\n\n[accent]Press the launch button.
+tutorial.pause = Du kannst das Spiel jederzeit [accent]pausieren.[]\nIn einer Pause kannst du den Bau weiterer Gebäude vorbereiten.\n\n[accent]Drücke die Leertaste um zu pausieren.
```
Comment (Cyprex) [inline core/assets/bundles/bundle_de.properties]: >During Battle you are able to pause the game

Your translation is roughly "You can pause the game anytime."

This is an improvement over the original, what's the stance on this?



> In einer Pause kannst du den Bau weiterer Gebäude vorbereiten.

Maybe "in Auftrag geben"?

**Comment by Anuken** [comment]
@intergalacticsuperhero I *could* merge it, but the changeset is so messed up that I can't tell what the difference is without going 80 unrelated commits. It would be best if you made a new PR, if that's not too inconvenient.

**Comment by intergalacticsuperhero** [comment]
> @intergalacticsuperhero I _could_ merge it, but the changeset is so messed up that I can't tell what the difference is without going 80 unrelated commits. It would be best if you made a new PR, if that's not too inconvenient.

Yes, sorry for that.
I have just created a new pull request based on the current master branch: #703 

This pull request can thus be closed :-)

**Pull request closed without merging**